kuma/landing/urls.py

```diff
@@ -23,6 +24,7 @@
         views.robots_txt,
         name='robots_txt'),
     url(r'^favicon.ico$',
-        views.FaviconRedirect.as_view(icon='favicon.ico'),
+        shared_cache_control(
+            views.FaviconRedirect.as_view(icon='favicon.ico')),
```

Comment:
Eventually, we'll want to cache endpoints that change rarely differently than endpoints that change often - this redirect would be one of them. It may make sense to have one decorator for rarely changing endpoints, and another for ones that change unpredictably. For now, it is probably OK to use the same cache header.